### ISSUE TITLE
Get tests working again on stable rust

### ISSUE THREAD

**Issue by kellpossible** [kellpossible/cargo-i18n#16]
Currently tests only work on nightly because of this error:

```
error[E0658]: procedural macros cannot be expanded to statements
  --> src/lib.rs:142:1
   |
16 | language_loader!(MyLanguageLoader);
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/54727

error: aborting due to previous error
```

which requires `#![feature(proc_macro_hygiene)]`. I'm not really sure why this only happens in some tests but not others which use this macro...

**Comment by kellpossible** [comment]
Looks like this is set to be solved with rust 1.45

**Comment by kellpossible** [comment]
Looks like this issue is no longer a problem now with Rust stable version `1.45`.
